src/site/content/en/blog/gamepad/index.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Play the Chrome dino game with your gamepad
+subhead: Learn how to use the Gamepad API to push your web games to the next level.
+description: |
+  This article teaches you how to use the Gamepad API to control web games.
+authors:
+  - thomassteiner
+date: 2020-10-30
+hero: hero.jpg
+alt: The hands of a person playing the Chrome Dino game on a Nintendo Switch.
+tags:
+  - blog # blog is a required tag for the article to show up in the blog.
+  - capabilities
+  - gamepad
+---
+
+Chrome's offline page easter egg is one of the worst-kept secrets in history
+(`[citation needed]`, but claim made for the dramatic effect).
+If you press the <kbd>space</kbd> key or, on mobile devices, tap the dinosaur,
+the offline page becomes an actually playable arcade game.
+You might be aware of the fact that you do not actually have to go offline
+when you feel like playing: You can also just navigate to `chrome://dino`, or,
+for the geek in you, browse to
+`chrome://network-error/-106`.
+But did you know that there are currently
+[270 million games played every month](https://www.blog.google/products/chrome/chrome-dino#jump-content:~:text=There%20are%20currently%20270%20million%20games%20played%20every%20month)?
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled"
+       src="dino.png"
+       alt="Chrome's offline page with the Chrome dino game.">
+  <figcaption class="w-figcaption">
+    Press the space bar to play!
+  </figcaption>
+</figure>
+
+Another fact that arguably is more useful to know and that you might not be aware of
+is that the game can be played with a gamepad.
+Gamepad support was added exactly one year ago as of the time of this writing in a
```

Comment:
Maybe mention that gamepad only works in arcade mode. (in other words if the URL is `chrome://dino`)

Reply:
Will do.